src/harberger-lib/components/Info.re

```diff
@@ -240,3 +241,168 @@ let make = (~tokenId: TokenId.t) => {
+          )
+        );
+
+    let ratio = QlHooks.usePledgeRate(tokenId);
+
+    let monthlyRate = Js.Float.toString(ratio *. 100.);
+
+    <React.Fragment>
+      <div>
+        {if (ratio == 0.) {
+           <p>
+             "The monthly pledge rate will be revealed at launch."->restr
+           </p>;
+         } else {
+           <>
+             <small>
+               <strong>
+                 "Monthly Pledge Rate:"->restr
+                 <Rimble.Tooltip
+                   message={
+                     "This is the monthly percentage contribution of "
+                     ++ tokenName
+                     ++ "'s sale price that will go towards conservation of at risk animals. This is deducted continuously from the deposit and paid by the owner of the animal"
+                   }
+                   placement="top">
+                   <span> {js|ⓘ|js}->restr </span>
+                 </Rimble.Tooltip>
+               </strong>
+             </small>
+             <br />
+             {(monthlyRate ++ " %")->restr}
+           </>;
+         }}
+      </div>
+      {abandoned
+         ? <p>
+             <strong>
+               "Was abandoned by "->restr
+               <a
+                 onClick={e => {
+                   ReactEvent.Mouse.preventDefault(e);
+                   clearAndPush({j|/#user/$currentPatron|j});
+                 }}>
+                 displayNameStr->restr
+               </a>
+               <Rimble.Tooltip
+                 message={j|This happens when the user's deposit runs out for the wildcard.|j}
+                 placement="top">
+                 <span> {js|ⓘ|js}->restr </span>
+               </Rimble.Tooltip>
+             </strong>
+             <br />
+           </p>
+         : <p>
+             "This wildcard has never had a guardian - you can be the first."
+             ->React.string
+           </p>}
+      <p>
+        <small>
+          <strong>
+            {{
+               tokenName ++ "'s Patronage: ";
+             }
+             ->restr}
+            <Rimble.Tooltip
+              message={
+                "This is the total contribution that has been raised thanks to the wildcard, "
+                ++ tokenName
+              }
+              placement="top">
+              <span> {js|ⓘ|js}->restr </span>
+            </Rimble.Tooltip>
+          </strong>
+        </small>
+        <br />
+        {{
+           totalPatronage ++ " ETH";
+         }
+         ->restr}
+        <br />
+        <small>
+          {{
+             "(" ++ totalPatronageUsd ++ " USD)";
+           }
+           ->restr}
+        </small>
+      </p>
+      {abandoned
+         ? <>
+             <p>
+               <small>
+                 <strong>
+                   "Abandoned since: "->restr
+                   <Rimble.Tooltip
+                     message={
+                       "This is the date the deposit will run out and the current owner will lose ownership of "
```

Comment:
Should it be 'ran' out here? when it's - abandoned. Also the end of the sentence should change then to current owner lost ownership etc

Reply:
Yeah, swapped them around. Good catch